VMBackup/main/fsfreezer.py

```diff
@@ -156,15 +157,21 @@ def freeze_safe(self,timeout):
                             self.logger.log(line.rstrip(), True)
                         else:
                             break
-                error_msg="freeze failed for some mount"
-                freeze_result.errors.append(error_msg)
-                self.logger.log(error_msg, True, 'Error')
+                if (sig_handle == 0):
+                    timedout = True
```

Comment:
if freeze fails and ends up in these flows, i dont think we are making a thaw call.
Then where are we setting "enforce_local_flag" to true. Can you please confirm
This might be existing code and we are getting the logs means, things are not even ending in this flow. 
I can only see it getting set in Thaw

Reply:
at the end it is anyways commited to local (end of enable)